week1/armstrong.rb

```diff
@@ -9,5 +9,17 @@
 # and `false` otherwise.
 
 def armstrong_number(n)
-    raise NotImplementedError # Delete this line and start coding here
+    if(n < 0)
+        return false
+    end
+    
+    x = n.to_s
+    ans = 0
+    x.each_char { |c|
+        ans += (c.to_i)**x.length
+    }
+    if(ans == x)
+        return true
+    end
+    return ans
```

Comment:
Our test case expects you to return `false` and not the number 🥲

Reply:
Ohh god, I was testing that out for debugging and left it there. Sorry!